docs/get-started/vue-vite.md

```diff
@@ -0,0 +1,268 @@
+
+First, install the framework:
+
+<!-- prettier-ignore-start -->
+
+<CodeSnippets
+  paths={[
+    'vue/vue3-vite-install.npm.js.mdx',
+    'vue/vue3-vite-install.pnpm.js.mdx',
+    'vue/vue3-vite-install.yarn.js.mdx'
+  ]}
+/>
+
+<!-- prettier-ignore-end -->
+
+Then, update your `.storybook/main.js|ts` to change the framework property:
+
+<!-- prettier-ignore-start -->
+
+<CodeSnippets
+  paths={[
+    'vue/vue3-vite-add-framework.js.mdx',
+    'vue/vue3-vite-add-framework.ts.mdx'
+  ]}
+/>
+
+<!-- prettier-ignore-end -->
+
+## Extending the Vue application
+
+Storybook creates a [Vue 3 application](https://vuejs.org/api/application.html#application-api) for your component preview.
+When using global custom components (`app.component`), directives (`app.directive`), extensions (`app.use`), or other application methods, you will need to configure those in the `./storybook/preview.js` file.
+
+Therefore, Storybook provides you with a `setup` function exported from this package, which receives as a callback your Storybook instance, which you can interact with and add your custom configuration.
+
+```js
+// .storybook/preview.js
+import { setup } from '@storybook/vue3';
+
+setup((app) => {
+  app.use(MyPlugin);
+  app.component('my-component', MyComponent);
+  app.mixin({
+    /* My mixin */
+  });
+});
+
+// Rest of the file...
+```
+
+## Using `vue-component-meta`
+
+[`vue-component-meta`](https://github.com/vuejs/language-tools/tree/master/packages/component-meta) is a tool that extracts metadata from Vue components. Storybook can use it to generate the [controls](../essentials/controls.md) for your stories and documentation. It's a more full-featured alternative to `vue-docgen-api` and is recommended for most projects.
+
+If you want to use `vue-component-meta`, you can configure it in your `.storybook/main.js|ts` file:
+
+```ts
+// .storybook/main.ts
+import type { StorybookConfig } from '@storybook/vue3-vite';
+
+const config: StorybookConfig = {
+  framework: {
+    name: '@storybook/vue3-vite',
+    options: {
+      docgen: 'vue-component-meta',
+    },
+  },
+};
+
+export default config;
+```
+
+`vue-component-meta` comes with many benefits and enables more documentation features, such as:
+
+### Support for multiple component types
+
+`vue-component-meta` supports from all types of Vue components, including functional, composition / options API components, and from `.vue`, `.ts`, `.tsx`, `.js`, and `.jsx` files.
+
+It also supports both default and named component exports.
+
+### Prop description and JSDoc tag annotations
+
+To provide a description for a prop, including tags, you can use JSDoc comments in your component's props definition:
+
+```html
+<!-- YourComponent.vue -->
+<script setup lang="ts">
+  interface MyComponentProps {
+    /**
+     * The name of the user
+     */
+    name: string;
+    /**
+     * The category of the component
+     * @required
+     * @default 'Uncategorized'
+     */
+    category: string;
+  }
```

Comment:
```suggestion
interface MyComponentProps {
  /**
   * The name of the user
   */
  name: string;
  /**
   * The category of the component
   *
   * @since 8.0.0
   */
  category?: string;
}

withDefaults(defineProps<MyComponentProps>(), {
  category: "Uncategorized",
});
```

Required will be automatically detected so for`category?: string` it will be false and true for `category: string`.
Also the default will probably be inferred by `withDefaults` for the most projects, right?